transports/quartz/src/main/java/org/mule/transport/quartz/QuartzMessageReceiver.java

```diff
@@ -138,13 +138,13 @@ protected void doStart() throws MuleException
 
             if (cronExpression != null)
             {
-                CronTrigger ctrigger = new CronTrigger();
+                CronTriggerImpl ctrigger = new CronTriggerImpl();
                 ctrigger.setCronExpression(cronExpression);
                 trigger = ctrigger;
             }
             else if (repeatInterval != null)
             {
-                SimpleTrigger strigger = new SimpleTrigger();
+                SimpleTriggerImpl strigger = new SimpleTriggerImpl();
```

Comment:
again the same as in the dispatcher. COnsider having factory methods to create these things